Cos3rd.java

```diff
@@ -1,13 +1,29 @@
 class Cos3rd extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * This program lets you enter two sides of a triangle and the contained angle, and that will use the cosine law to calculate the third side 
+  * @author: A. Emmanuel-Singh
   */
   
   public void run() {
     
-    // start coding here
-    
+    // Declare Variable 
+    double dblSide1;
+    double dblSide2;
+    double dblAngle;
+    double dblSide3;
+
+    // Get value 
+    dblSide1 = readDouble("Input the lenth of the the first side of the triangle: ");
+    dblSide2 = readDouble("Input the length of the second side of the triangle: ");
+    dblAngle = readDouble("Enter the degree of the angle in degrees: ");
+
+    // Calaculate 
+    dblAngle = Math.toRadians(dblAngle);
+    dblSide3 = Math.sqrt((dblSide1 * dblSide1) + (dblSide2 * dblSide2) - (2 * dblSide1 * dblSide2 * Math.cos(dblAngle)));
```

Comment:
Use "math.pow" instead of "dblSide2 * dblSide2"  

Both lines of code can be combined to: dblArea = (dblSideA * dblSideB * Math.sin(Math.toRadians(dblAngleC))) / 2.0;

but both methods still work